modules/services/window-managers/i3-sway/i3.nix

```
@@ -143,50 +143,47 @@ let
-      force_focus_wrapping ${if focus.forceWrapping then "yes" else "no"}
-      focus_follows_mouse ${if focus.followMouse then "yes" else "no"}
-      focus_on_window_activation ${focus.newWindow}
-      mouse_warping ${if focus.mouseWarping then "output" else "none"}
-      workspace_layout ${workspaceLayout}
-      workspace_auto_back_and_forth ${
-        if workspaceAutoBackAndForth then "yes" else "no"
-      }
-
-      client.focused ${colorSetStr colors.focused}
-      client.focused_inactive ${colorSetStr colors.focusedInactive}
-      client.unfocused ${colorSetStr colors.unfocused}
-      client.urgent ${colorSetStr colors.urgent}
-      client.placeholder ${colorSetStr colors.placeholder}
-      client.background ${colors.background}
-
-      ${keybindingsStr { keybindings = keybindingDefaultWorkspace; }}
-      ${keybindingsStr { keybindings = keybindingsRest; }}
-      ${keycodebindingsStr keycodebindings}
-      ${concatStringsSep "\n" (mapAttrsToList (modeStr false) modes)}
-      ${concatStringsSep "\n" (mapAttrsToList assignStr assigns)}
-      ${concatStringsSep "\n" (map barStr bars)}
-      ${optionalString (gaps != null) gapsStr}
-      ${concatStringsSep "\n" (map floatingCriteriaStr floating.criteria)}
-      ${concatStringsSep "\n" (map windowCommandsStr window.commands)}
-      ${concatStringsSep "\n" (map startupEntryStr startup)}
-      ${concatStringsSep "\n" (map workspaceOutputStr workspaceOutputAssign)}
+      ${concatStringsSep "\n" ([
+        (fontConfigStr fonts)
+        "floating_modifier ${floating.modifier}"
+        (windowBorderString window floating)
+        "hide_edge_borders ${window.hideEdgeBorders}"
+        "force_focus_wrapping ${if focus.forceWrapping then "yes" else "no"}"
+        "focus_follows_mouse ${if focus.followMouse then "yes" else "no"}"
+        "focus_on_window_activation ${focus.newWindow}"
+        "mouse_warping ${if focus.mouseWarping then "output" else "none"}"
+        "workspace_layout ${workspaceLayout}"
+        "workspace_auto_back_and_forth ${
+          if workspaceAutoBackAndForth then "yes" else "no"
+        }"
+        "client.focused ${colorSetStr colors.focused}"
+        "client.focused_inactive ${colorSetStr colors.focusedInactive}"
+        "client.unfocused ${colorSetStr colors.unfocused}"
+        "client.urgent ${colorSetStr colors.urgent}"
+        "client.placeholder ${colorSetStr colors.placeholder}"
+        "client.background ${colors.background}"
+        (keybindingsStr { keybindings = keybindingDefaultWorkspace; })
+        (keybindingsStr { keybindings = keybindingsRest; })
+        (keycodebindingsStr keycodebindings)
+      ] ++ mapAttrsToList (modeStr false) modes
+        ++ mapAttrsToList assignStr assigns ++ map barStr bars
+        ++ optional (gaps != null) gapsStr
+        ++ map floatingCriteriaStr floating.criteria
+        ++ map windowCommandsStr window.commands ++ map startupEntryStr startup
```

```suggestion
        ++ map windowCommandsStr window.commands
        ++ map startupEntryStr startup
```

Will the formatter approve of this change?